frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/icons.tsx

```diff
@@ -15,6 +15,16 @@ export type ColoredIconProps = {
   size?: 'sm' | 'md' | 'lg' | 'xl';
 };
 
+/**
+ * Component for displaying a green check mark circle icon
+ * @param {string} [className] - (optional) class name for styling
+ * @param {string} [title] - (optional) icon title
+ * @param {string} [size] - (optional) icon size
```

Comment:
Go ahead and list the options here  'sm' | 'md' | 'lg' | 'xl' and for any place with the size param